.github/ISSUE_TEMPLATE/1_bug_report_form.yml

```diff
@@ -0,0 +1,75 @@
+name: Bug report
+description: Create a report to help us improve
+title: "[BUG]: "
+labels: [triage needed, bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to report a bug in the KlinicCon-Frontend project.
+  - type: markdown
+    attributes:
+      value: |
+        Before filing a new issue, **please do a quick search** to check that it hasn't already been filed on the issue tracker. You can do this by going to [this link](https://github.com/rt4914/KlinicCon-Frontend/issues) and typing some words related to the issue in the search box next to the "New issue" button.
+  - type: textarea
+    id: issue-description
+    attributes:
+      label: Describe the bug
+      description: A concise description of what issue you're experiencing. **Example:** "When I click on the first lesson, I expect to be redirected to the lesson player, but instead the app crashes and shuts down."
+    validations:
+      required: true
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior. **Example:** "1. Go to the topic page for Place Values. 2. Click on the first lesson. 3. See that the app shuts down."
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: A clear and concise description of what you expected to happen. **Example:** "I expect to be redirected to the lesson player screen."
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots-or-videos
+    attributes:
+      label: Screenshots/Videos
+      description: |
+        If applicable, add screenshots or videos to help explain your problem.
+
+        **Tip:** You can attach images or log files by clicking this area to highlight it and then dragging files in.
+    validations:
+      required: false
+  - type: input
```

Comment:
Update this entire section to get browser and device information like 
Macbook Pro M1
Chrome xx.yy.zz version
etc.

Reply:
resolved. thanks.